docs/source/index.md

```diff
@@ -59,17 +59,17 @@ Feel free to report any issues/missing information so we can take a look into it
 :maxdepth: 2
 
 intro
-capabilities
-jupyter/sdk.ipynb
+apps/quickstart
 ```
 
 ```{toctree}
 :hidden: true
 :caption: MarketPlace Applications
 :maxdepth: 2
 
-apps/quickstart
 apps/mp-api
+capabilities
```

Comment:
If the file is here, I would consider moving it to the `apps` subfolder

Reply:
Done.